gradle/validation/error-prone.gradle

```diff
@@ -67,43 +67,45 @@ allprojects { prj ->
 
             // List of enabled/disabled checks
             // Please keep this synced with https://errorprone.info/bugpatterns when upgrading!
+            // Do *NOT* enable checks based on their name or description. Read the source code and make sure they are useful!
+            // Most error-prone checks are not useful for non-google software.
 
             // On by Default : ERROR
```

Comment:
Why have "On" by default if "most error-prone checks are not useful"?
Based on what I see below, if commenting makes it go away, then we are not "on" by default, thus perhaps this comment is wrong.

Reply:
the comment refers to the "On by default : ERROR" section of this page: https://errorprone.info/bugpatterns

the idea is to organize our (grouping, order) checks consistent with that page and to update it when updating the error-prone.

Reply:
aaaah.  I thinking updating these comments like this would be more clear:
// Section "On by Default : ERROR"

and maybe a separate comment that we've disabled all checks by default, overriding the error-prone default setting.  At least I'm assuming we've done this.

Reply:
i think it already says all this. click "view file" because github isn't showing you enough context?

Reply:
Indeed; thanks :-)